docs/index.rst

```diff
@@ -6,44 +6,39 @@ This is the central page for all PyDy's Documentation.
 
 .. include:: ../README.rst
 
-codegen package
----------------
+system module
+=============
 
 .. toctree::
-    :maxdepth: 2
+   :maxdepth: 1
 
-    codegen/api.rst
+   system.rst
 
-System class
-------------
+models module
+=============
 
 .. toctree::
    :maxdepth: 1
 
-   system.rst
+   models.rst
 
-viz package
------------
+codegen package
+===============
 
 .. toctree::
     :maxdepth: 2
```

Comment:
These toctrees for each of these packages/modules confuse me because they make me think that the subsections have their own page, when in fact they are just subheadings on the package's or module's page.